win32/src/winapi/stack_args.rs

```diff
@@ -49,6 +49,30 @@ impl<'a, T: FromArg<'a>> FromStack<'a> for T {
     }
 }
 
+impl<'a> FromArg<'a> for u8 {
```

Comment:
I wonder if we could get away with `FromArg<T> where T: Pod`.  I guess it's only Pod <32 bits though.  This is fine too.